packages/ui-components/src/components/LoadingIndicator/LoadingIndicator.component.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+import Loading from "./loading-indicator.svg"
+
+export interface LoadingIndicatorProps {
+  /**
+   * The size of the LoadingIndicator in pixels.
+   */
+  size?: string
```

Comment:
Few questions:
- Should we allow `size` as a `number` as well and account for that in the code? Is there a reason it's only a `string`?
- We expect the `size` to be passed as a `string`, without `px` included. What if the user does include `px` (or something else entirely) as part of the input? Is that fine? Or should we filter it out? Allowing only `number` type would really help here (if we indeed only want that)... but I understand it's not so easy to change it. Would just love to understand the constraints so we can cover edge cases here. 💪 Maybe I misunderstood something though. 

CC: @edda

Reply:
Those are good questions. I can't tell you why we made the choice of going with string here. I agree that going with number type instead would take care of a bunch of edge cases because people can only put numbers in, so there'll never be the edge case of people accidentally passing the unit together with the number.

The only thing I can think of is that the svg component requires the width/height to be a string but that can easily be solved by turning the passed number into a string. So I agree with you, let's make it number instead of string.

Reply:
Perfect, Thank You so much. Noted it here: https://github.com/cloudoperators/juno/issues/234#issuecomment-2435175493 and I'll create a ticket for future improvements, as discussed. 🙏